server/routes/prisons/sessionTemplates/addSessionTemplateController.ts

```diff
@@ -11,20 +11,23 @@ export default class AddSessionTemplateController {
     private readonly prisonService: PrisonService,
     private readonly sessionTemplateService: SessionTemplateService,
     private readonly incentiveGroupService: IncentiveGroupService,
+    private readonly categoryGroupService: CategoryGroupService,
   ) {}
 
   public view(): RequestHandler {
     return async (req, res) => {
       const { prisonId } = req.params
       const prison = await this.prisonService.getPrison(res.locals.user.username, prisonId)
       const incentiveGroups = await this.incentiveGroupService.getIncentiveGroups(res.locals.user.username, prisonId)
+      const categoryGroups = await this.categoryGroupService.getCategoryGroups(res.locals.user.username, prisonId)
```

Comment:
This is fine but because these three 'await' requests don't depend on each other they could potentially be put inside a Promise.all() so they run together. Could do it when location groups added. Although perhaps overkill as we've not got high traffic or poor performance!